WordPress/Classes/ViewRelated/MeViewController.swift

```diff
@@ -300,6 +303,9 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
     // MARK: - Notification observers
 
     func refreshModelWithNotification(notification: NSNotification) {
+        // Regenerate the header view if the account has changed
+        cachedHeaderView = MeHeaderView()
```

Comment:
I think I was seeing an issue when logging out and back in where the previous account details would be displayed until the new ones had been loaded. This line means that on logout the header view is instantly replaced with the generic account icon / name. I can check this though!